### PR TITLE
Creating dynamic form using ConnectorProperty

### PR DESCRIPTION
**Pull request by uidoyen**
- Created dynamic form using connector properties
- Upgraded Formik
- Added Switch Component
- Added Formik to nohoist

### CODE REVIEW

File: ui/packages/ui/src/app/pages/createConnector/CreateConnectorPage.tsx [hunk 1]
```diff
@@ -147,6 +147,7 @@ export const CreateConnectorPage: React.FunctionComponent = () => {
     propertyValues: Map<string, string>,
     category: PropertyCategory
   ): void => {
+    console.log(propertyValues)
```
Comment (mdrillin) [inline ui/packages/ui/src/app/pages/createConnector/CreateConnectorPage.tsx:150]: please remove the console.log

**Review by mdrillin — APPROVED** [review]
I'm good with your changes.  console.log was giving a build error - pls remove that then you can merge it.

File: ui/packages/ui/tslint.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
   },
   "rules": {
     "jsx-no-lambda": false,
+    "ordered-imports": false,
```
Comment (indraraj) [inline ui/packages/ui/tslint.json:10]: I think we could leave the ordered-import true, as vscode(lint plugins) itself provides a one-click fix for it.

**Review by indraraj — APPROVED** [review]
one suggestion in the tslint.json file rest looks good to me.

**Pull request merged**